### ISSUE TITLE
Reading Service Signals from CANoe

### ISSUE THREAD

**Issue by fabioreway** [chaitu-ycr/py_canoe#49]
Hi, is it possible to get values from service signals, for example, from some/ip services events?
Thanks

**Comment by JeganJayaraj** [comment]
Yes, As per the Vector CANoe documentation. It is possible to get/set the service signal values.

**Comment by chaitu-ycr** [comment]
hi @JeganJayaraj 
i couldn't find any references to get/set the service signal values.
can you please help me in with path / share screenshot of document.
